.pre-commit-config.yaml

```diff
@@ -16,13 +16,13 @@ repos:
     - id: helm-docs
       name: helm-docs
       language: docker_image
-      entry: jnorwood/helm-docs:latest --chart-search-root=diracx --output-file=../README.md --template-files=../README.md.gotmpl
+      entry: jnorwood/helm-docs:v1.11.2 --chart-search-root=diracx --output-file=../README.md --template-files=../README.md.gotmpl
```

Comment:
Any particular reason ?

Reply:
Mostly so that the hook is stable. The CI pulls the actual latest where-as locally I get `jnorwood/helm-docs:v1.11.0` as "latest". We should just have a fixed version everywhere.